src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistry.java

```diff
@@ -116,6 +117,17 @@ public <JavaT, CqlT> JSONCodec<JavaT, CqlT> codecToCQL(
               CollectionCodecs.buildToCQLSetCodec(valueCodecCandidates, st.getElementType());
         }
         // fall through
+      } else if (columnType instanceof VectorType vt) {
```

Comment:
https://cassandra.apache.org/doc/latest/cassandra/developing/cql/types.html#collections
One thing is interesting, c* does not call vector as a collection column dataType.
But vector is not a scalar.
So it is kind of stuck in the middle.

Reply:
Yes, need to keep that in mind wrt terminology. LMK if I missed some reference, calling vectors collections?